config/agent/agent.conf

```diff
@@ -11,8 +11,9 @@
 NodeName=
 
 #
-# The IP address that hirte-agent can use to connect to hirte manager. It must be a valid IPv4 or IPv6 address.
-# It's mandatory to set either ManagerHost or ManagerAddress option for each hirte-agent.
+# The IP address that hirte-agent can use to connect to hirte manager. It must
+# be a valid IPv4 or IPv6 address. One of the ManagerHost or ManagerAddress
+# options must be set for each hirte-agent.
 ManagerHost=
```

Comment:
II intentionally put sentence `It's mandatory to set either ManagerHost or ManagerAddress option for each hirte-agent.` to its own line to be more visibile. But if you think it's visible enough ...

Reply:
I actually question this, and plan on opening another PR to make mangerhost and manageraddress default to localhost. I would like to not be required to set any of these fields

Reply:
Does it really makes sense to set ManagerHost to `localhost` when hirte-agent is mainly intended to be used on remote node? Yes, it will help with the hirte-agent installation on the manager node, but on the other nodes the error would be more confusing: I mean hirte-agent will fail with something like `Cannot connect to localhost` instead of `ManagerHost or ManagerAdress need to be specified`?

P.S. I'm not saying we are failing with exactly those errors :-)

Reply:
Currently if you don't configure you fail with those errors.  I am thinking that almost everyone will have a hirte-agent on the same box as the hirte main, and those should just work.

Reply:
I'm resolving this discussion since I guess it will continue on the change that @rhatdan is proposing